sys/xtimer/xtimer.c

```diff
@@ -58,15 +58,15 @@ void _xtimer_sleep(uint32_t offset, uint32_t long_offset)
     mutex_lock(&mutex);
 }
 
-void xtimer_periodic_wakeup(uint32_t *last_wakeup, uint32_t period) {
+void _xtimer_periodic_wakeup(uint32_t *last_wakeup, uint32_t period) {
     xtimer_t timer;
     mutex_t mutex = MUTEX_INIT;
 
     timer.callback = _callback_unlock_mutex;
     timer.arg = (void*) &mutex;
 
     uint32_t target = (*last_wakeup) + period;
-    uint32_t now = xtimer_now();
+    uint32_t now = _xtimer_now();
```

Comment:
You need the underscore variant on L106 xtimer_spin() and L115 xtimer_now() as well